codeception.yml

```diff
@@ -25,34 +29,48 @@ suites:
         class_name: SprykTester
         modules:
             enabled:
-                - SprykerSdkTest\Module\DevelopmentModeModule:
-                    developmentMode: 'core'
+                - Symfony:
+                      part: services
+                      app_path: 'src'
```

Comment:
Paths should ideally always have trailing slash /

Reply:
with slash ./src /src it always goes to ../src 
![image](https://user-images.githubusercontent.com/5063777/159224241-5245ffd4-a1df-4a2c-872a-48dad672227d.png)